src/app/users/page.tsx

```
@@ -27,10 +33,10 @@ export default function UsersPage() {
   }, [status, router]);
 
   useEffect(() => {
-    if (session?.accessToken) {
+    if (canViewUsers && session?.accessToken) {
       fetchUsers();
```

ESLint

[error] Error: Cannot access variable before it is declared

`fetchUsers` is accessed before it is declared, which prevents the earlier access from updating when this value changes over time.

/home/jailuser/git/src/app/users/page.tsx:37:7
  35 |   useEffect(() => {
  36 |     if (canViewUsers && session?.accessToken) {
> 37 |       fetchUsers();
     |       ^^^^^^^^^^ `fetchUsers` accessed before it is declared
  38 |     }
  39 |   }, [canViewUsers, session]);
  40 |

/home/jailuser/git/src/app/users/page.tsx:41:3
  39 |   }, [canViewUsers, session]);
  40 |
> 41 |   const fetchUsers = async () => {
     |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 42 |     try {
     | ^^^^^^^^^
> 43 |       const response = await fetch('/api/devops/users');
     …
     | ^^^^^^^^^
> 57 |     }
     | ^^^^^^^^^
> 58 |   };
     | ^^^^^ `fetchUsers` is declared here
  59 |
  60 |   // Get unique email domains for filter dropdown
  61 |   const emailDomains = Array.from(

(react-hooks/immutability)